src/ua.c

```diff
@@ -1239,6 +1239,62 @@ int ua_debug(struct re_printf *pf, const struct ua *ua)
 }
 
 
+/**
+ * Print the user-agent information in JSON
+ *
+ * @param od  User-Agent dict
+ * @param ua  User-Agent object
+ *
+ * @return 0 if success, otherwise errorcode
+ */
+int ua_state_json_api(struct odict *od, const struct ua *ua)
+{
+	struct odict *reg = NULL;
+	struct odict *cfg = NULL;
+	struct le *le;
+	int err = 0;
+
+	if (!ua)
+		return 0;
+
+	err |= odict_alloc(&reg, 8);
+	err |= odict_alloc(&cfg, 8);
+
+	/* user-agent info */
+	err |= odict_entry_add(od, "cuser", ODICT_STRING, ua->cuser);
+	/* TODO: 'selected_ua' not working, it's always true? */
+	err |= odict_entry_add(od, "selected_ua", ODICT_BOOL,
+			ua == uag_current());
```

Comment:
this logic is working for me, when using the command line interface and 2 accounts.

```
"selected_ua":true,"aor":"sip:aeh@iptel.org"
"selected_ua":false,"aor":"sip:alfredh@sip.antisip.com"
```

Reply:
That is true! I must have got the wrong result since I did not append it to the array correctly. or just read it wrong.